tmt/steps/prepare/shell.py

```diff
@@ -81,6 +121,35 @@ def go(
         workdir = self.step.plan.worktree
         assert workdir is not None  # narrow type
 
+        if self.data.url:
+            repo_path = workdir / "repository"
+
+            self._env_var = str(repo_path.resolve())
```

Comment:
I'm not sure what is the purpose of this line, it would overwrite the environment variable name with repository URL. That does not look like intended action.

Reply:
Sorry, indeed I misunderstood the idea, I think now it's correct